packages/table/src/regions.ts

```diff
@@ -548,11 +541,52 @@ export class Regions {
         }
     }
 
+    private static overlapsRegionInternal(regions: IRegion[], query: IRegion, allowPartialOverlap = true) {
```

Comment:
👎 why can't this just be `overlapsRegion`? the third prop is optional with a default anyway. 
i don't see the need for both.